addons/nuxeo-template-rendering/nuxeo-template-rendering-core/src/main/java/org/nuxeo/template/processors/AbstractBindingResolver.java

```diff
@@ -195,4 +276,17 @@ protected String getHtmlValue(BlobHolder bh) throws IOException {
         return "";
     }
 
+    protected class ValueNotFound extends Throwable {
```

Comment:
Should be `protected static class`. Please extend `NuxeoException` instead of `Throwable`. Same for `NoValueToAddInContext`.

Reply:
Ok I put the static.

As I told to Kevin, this is internal exception never thrown outside. Do you think this is required